conf/README.md

```diff
@@ -53,24 +55,28 @@ copy to the corresponding `.dist` file for this.
 
 ## Running webwork2 for development
 
-Make sure that `$server_root_url` in site.conf is set to `http://localhost:3000`.  After any other changes in the
-initial configuration of webwork2 you are ready to run webwork2 for development.  To do so from the webwork2 directory
-execute the following
+There are two important settings that you may need to change in `site.conf`
+
+* make sure that `$server_root_url` is set to `http://localhost:3000`.
+* make sure that `$pg_dir` is set to the top of your pg directory.
```

Comment:
Fix punctuation and capitilization.
```suggestion
There are two important settings that you may need to change in `site.conf`.

* Make sure that `$server_root_url` is set to `http://localhost:3000`.
* Make sure that `$pg_dir` is set to the top of your pg directory.
```

Also, if you are adding the comment about making sure that `$pg_dir` is set in `site.conf`, then a comment regarding setting `pg_dir` to the same thing in `webwork2.mojolicious.yml` needs to be made also.  Although, I will probably remove the `site.conf` setting, and make it so that is set from the setting in the `yml` file.

Reply:
It would be better to not have to change the same variable in multiple places, so the comment about `site.conf` would not be relevant.